frontend/packages/console-shared/src/components/health-checks/HealthChecksAlert.tsx

```diff
@@ -17,13 +17,15 @@ import {
   DaemonSetModel,
   StatefulSetModel,
 } from '@console/internal/models';
+import { useUserSettingsCompatibility } from '@console/shared/src/hooks/useUserSettingsCompatibility';
 import { STORAGE_PREFIX } from '../../constants';
 import './HealthChecksAlert.scss';
 
 type HealthChecksAlertProps = {
   resource: K8sResourceKind;
 };
 
+const HEALTH_CHECK_CONFIGMAP_KEY = 'console.healthChecks';
```

Comment:
`console` is used across for CM key prefix , will it make sense to keep in in const ?

Reply:
Moved.